src/foundry/client/pixi/core/loader.d.mts

```diff
@@ -14,7 +21,7 @@ declare global {
     /**
      * Initialize the basis transcoder for PIXI.Assets
      */
-    static initializeBasisTranscoder(): Promise<unknown>;
+    static initializeBasisTranscoder(): ReturnType<typeof TranscoderWorker.loadTranscoder> | Promise<void>;
```

Comment:
Nit; can you make `ReturnType<typeof TranscoderWorker.loadTranscoder>` a helper type instead? I'm trying to move away from "complex" expressions and into reusable types.